edu.wpi.first.wpilib.plugins.cpp/resources/templates/examples/Gyro/src/Robot.cpp

```diff
@@ -1,81 +1,42 @@
 #include <AnalogGyro.h>
-#include <CANTalon.h>
 #include <Joystick.h>
 #include <RobotDrive.h>
-#include <SampleRobot.h>
+#include <IterativeRobot.h>
```

Comment:
The WPILib headers should be in lexographic order. `#include <cmath>` should go before them rather than `#include <math.h>` going after.